book/yaml-referenz.md

```diff
@@ -12,7 +12,50 @@ YAML-Format: Eine Übersicht
 Yaml-Datei-Felder
 ------------------
 
-(lang, comment, product-id etc.)
+Um eine Yaml-Datei korrekt zu schreiben sind einige Strukturen einzuhalten.
+Auf diese Strukturen möchte ich nun etwas näher eingehen.
+
+Die Grundstruktur besteht dabei aus den Feldern
+<ul>
+<li>media-path:</li>
+<li>language:</li>
+<li>welcome:</li>
+<li>product-id:</li>
+<li>comment:</li>
+<li>init:</li>
+<li>scripts:</li>
+<li>speak:</li>
+</ul>
+
+Diese Felder möchte ich nun einzeln erläutern.
+<h3>media-pathy:</h3>
+Im diesem Feld wird vorgegeben, wo sich die Audiofiles zum Programm befinden. Der Ordner wird hier inkremental angegeben. Wird hier beispielsweise "media/test_%s" geschrieben, so wird der String "%s" durch den Namen im Programm ersetzt.
+Beispiel: Ein Aufruf mit P(audio) greift auf die Datei media/test_audio.ogg zurück.
+
+<h3>language:</h3>
+Wie der Name hier bereits vermuten lässt, lässt sich hier die Sprache vorgeben. Für Deutsch wählen wir "de".
```

Comment:
Das ist die Sprache für den Tiptoi-Stift; aber nicht die Sprache fürs text-to-speech.